SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/SpatialWorkerConnection.cpp

```diff
@@ -219,6 +219,21 @@ void USpatialWorkerConnection::ProcessLoginTokensResponse(const Worker_Alpha_Log
 	ConnectToLocator(&  DevAuthConfig);
 }
 
+void USpatialWorkerConnection::ReuqestDeploymentLoginTokens()
+{
+	Worker_Alpha_LoginTokensRequest LTParams{};
+	FTCHARToUTF8 PlayerIdentityToken(*DevAuthConfig.PlayerIdentityToken);
+	LTParams.player_identity_token = PlayerIdentityToken.Get();
+	FTCHARToUTF8 WorkerType(*DevAuthConfig.WorkerType);
+	LTParams.worker_type = WorkerType.Get();
+	LTParams.use_insecure_connection = false;
```

Comment:
This would ideally be configurable to support things like running locally

Reply:
Could you describe more detail about this? I don't get the main idea. Just combine previous codes into a new function.

Reply:
I should be able to set this to `true` if needed (possibly via `DevAuthConfig` or some other means)

Reply:
Sound reasonable, however it is not purpose of this ticket. Maybe could create a new ticket in UNR's backlog to  track  this.

Reply:
I agree with raymon. It would be nice to have, but slightly out of scope for this. Let's create an UNR ticket to get this tracked :)

Reply:
Yes, definitely, it wasn't something to add to this ticket. As long as it's tracked and prioritized at some point, it's all good. :)